generate-config.sh

```diff
@@ -540,6 +635,112 @@ setup_registry_credentials() {
     echo "${image_credentials}"
 }
 
+setup_ingress() {
+    url="$1"
+    ingress="{}"
+
+    hostname=$(echo ${url} | awk -F[/:] '{print $4}')
+
+    response=$(prompt "Use Kubernetes Ingress?" "${regex_yn}" "true" "No")
+    if [[ "${response}" =~ ^[yY] ]]; then
+        ingress=$(add_kv_bool_to_object "enabled" true "${ingress}")
+        hostname=$(prompt "Ingress Hostname" "" "true" "${hostname}")
+        hostname="[\"${hostname}\"]"
+        ingress=$(add_json_to_object "hosts" "${hostname}" "${ingress}")
+        ingress_class=$(prompt "Ingress Class" "" "true" "nginx")
```

Comment:
```suggestion
        ingress_class=$(prompt "Ingress Class (Optional)" "" "true")
        if [[ -n “${ingress_class}” ]]; then
            ingress=$(add_kv_to_object “className” “${ingress_class}” “${ingress}”)
        fi
```

Reply:
I wrote this on my phone - wanted to check it before committing

Reply:
we can put it in a follow-up, I think the quotes are smart quotes :laughing:

Reply:
Ha, just got home and fixed it

Reply:
:+1: